src/umpire/strategy/DynamicPool.hpp

```diff
@@ -31,129 +30,116 @@ namespace strategy {
  * and the minimum size controls the lower bound on all future chunk
  * allocations.
  */
-class DynamicPool :
-  public AllocationStrategy
+class DynamicPool : public AllocationStrategy
 {
   public:
+    using Pointer = void*;
+
     /*!
-     * \brief Callback Heuristic to trigger coalesce of free blocks in pool.
+     * \brief Callback heuristic to trigger coalesce of free blocks in pool.
      *
      * The registered heuristic callback function will be called immediately
      * after a deallocation() has completed from the pool.
      */
-    using Coalesce_Heuristic = std::function<bool( const strategy::DynamicPool& )>;
+    using CoalesceHeuristic = std::function<bool (const strategy::DynamicPool&)>;
 
     /*!
      * \brief Construct a new DynamicPool.
      *
-     * \param name Name of this instance of the DynamicPool.
-     * \param id Id of this instance of the DynamicPool.
-     * \param min_initial_alloc_size The minimum size of the first allocation
-     *                               the pool will make.
-     * \param min_alloc_size The minimum size of all future allocations.
-     * \param coalesce_heuristic Heuristic callback function.
+     * \param name Name of this instance of the DynamicPool
+     * \param id Unique identifier for this instance
+     * \param initial_alloc_bytes Size the pool initially allocates
+     * \param min_alloc_bytes The minimum size of all future allocations
+     * \param align_bytes Number of bytes with which to align allocation sizes
+     * \param coalesce_heuristic Heuristic callback function
```

Comment:
Please add description of `align_bytes` parameter.

Reply:
There was already one there, above `coalesce_heuristic`. I moved it below that so it's in order.